charts/opentelemetry-demo/grafana-dashboards/demo-dashboard.json

```diff
@@ -431,7 +438,7 @@
         },
         "showHeader": true
       },
-      "pluginVersion": "10.1.2",
+      "pluginVersion": "10.4.3",
```

Comment:
Should we pin the Grafana image version in the Chart values? Noticed that we are specifying a different version that the one being used https://github.com/open-telemetry/opentelemetry-helm-charts/blob/164410617e88d6d7d841deecb8db3fc3299c98e6/charts/opentelemetry-demo/examples/public-hosted-ingress/rendered/grafana/role.yaml#L12

Reply:
This dashboard is emitted by Grafana when you export it as JSON. That version is set by Grafana itself. We don't explicitly do it.

Reply:
And the Grafana version from where the dashboard was exported was `10.1.2`? (The one being deployed by the Chart)

Reply:
This JSON is a direct copy from the [demo repo](https://github.com/open-telemetry/opentelemetry-demo/blob/main/src/grafana/provisioning/dashboards/demo/demo-dashboard.json).

To create it, we build the dashboard exactly as needed in Grafana, export it from the Grafana UI, and use the resultant JSON as the dashboard definition that was updated in this [PR](https://github.com/open-telemetry/opentelemetry-demo/pull/1669).